pkg/crypto/sha256/LICENSE

```diff
@@ -1,202 +0,0 @@
-
-                                 Apache License
-                           Version 2.0, January 2004
-                        http://www.apache.org/licenses/
-
```

Comment:
Don't remove the 'pkg/crypto/sha256' we need to fix it,  remove the references of its usage.